protobuf/proto/crux.proto

```diff
@@ -558,6 +561,13 @@ message UpdateNodeRequest {
   string name = 100;
   optional string description = 101;
   optional string icon = 102;
+  optional NodeType type = 103;
```

Comment:
why optional?

Reply:
The user does not select on create or update node options. This could be removed from the Create / Update requests, I will do that.